src/platform/plugins/shared/usage_collection/server/usage_counters/usage_counters_service.test.ts

```diff
@@ -222,9 +224,16 @@ describe('UsageCountersService', () => {
     jest.runOnlyPendingTimers();
 
     // wait for retries to kick in on next scheduler call
-    await tick();
+    await tickWithDelay(100);
     // number of incrementCounter calls + number of retries
-    expect(mockIncrementCounter).toBeCalledTimes(2 + 1);
+    expect(mockIncrementCounter).toBeCalledTimes(2 + retryConst);
+    // expect(logger.warn).toHaveBeenNthCalledWith(1, 'Store counters into savedObjects', {
```

Comment:
Since we're running it with retryCount: 1, I'd expect this log to occur here.

Is there a regression?

Reply:
scratch that, I'd commented out the log. It's back now and does assert the log appears (https://github.com/elastic/kibana/pull/206363/commits/44724bbd73748ca4aaba521fb1a72fdfbfb7ec18)